### DISCUSSION TITLE
daemon directory location

### DISCUSSION THREAD

**Discussion by ncd76** [coreybutler/node-windows#341] [General]
Would it be possible to have the daemon subdirectory created in another folder like the users /temp folder?

I'm trying to package this up in pkg as to run the service to user needs node to start with - or - do you have an example of how to create and exe for windows that starts a node.js service running?

Thx...

**Comment by coreybutler** [comment]
You should either use node-windows OR pkg, not both. Like node-windows, pkg produces a binary, but it bundles Node in the binary. Pkg is meant to produce completely standalone executables at the cost of large output (~67MB + script size). It also means the developer has to manage the background process. Node-windows generates smaller output (58Kb + script) since Node is intentionally _not_ bundled with the binary, at the cost of expecting Node to be installed.

If you need a standalone executable, you might be able to use some of the wrapper source code from node-windows directly in your own scripts to manage the process, then use pkg to produce a standalone executable. I haven't done that personally, so your mileage may vary.
